spyder/plugins/editor/widgets/tests/test_codeeditor.py

```diff
@@ -56,3 +56,47 @@ def test_editor_lower_to_upper(editorbot):
     widget.transform_to_uppercase()
     new_text = widget.get_text('sof', 'eof')
     assert text != new_text
+
+def test_editor_remove_empty_line(editorbot):
```

Comment:
Please add two blank lines of separation between tests. That's why pep8speaks is still complaining.